docs/site/reference/formatter-spec.md

```diff
@@ -60,3 +60,54 @@ outputs.
 ### 4. Reliable
 
 We expect the formatter to be reliable and not break the semantics of the formatted files.
+
+## Stdin Specification
```

Comment:
Should I actually promote this to a top level file? That is, `docs/site/reference/stdin-spec.md`?